### PR TITLE
BUG 1671528: QE induced tweaks to recent new-app CRD error message

### PR DESCRIPTION
**Pull request by gabemontero**
/assign @adambkaplan

### CODE REVIEW

**Comment by gabemontero** [comment]
@soltysh some minor tweaks based on QE feedback in https://bugzilla.redhat.com/show_bug.cgi?id=1671528

**Comment by gabemontero** [comment]
@adambkaplan might also start working on updating OWNERS for devex CLI paths like new-app

**Comment by gabemontero** [comment]
aws vpc quota pain for e2e-aws-serial

**Comment by gabemontero** [comment]
aws vpc quota pain e2e-aws-serial 

/retest

File: pkg/oc/cli/newapp/newapp.go [hunk 1]
```diff
@@ -1020,9 +1020,14 @@ func TransformRunError(err error, baseName, commandName, commandPath string, gro
...
-			useProcessMsg := fmt.Sprintf("The template contained an object type unknown to `oc new-app`.  Use `oc process | oc create` instead.  Error details: %v", err)
-			if len(config.ComponentInputs.TemplateFiles) > 0 {
+			var useProcessMsg string
+			switch {
+			case len(config.ComponentInputs.TemplateFiles) > 0:
 				useProcessMsg = fmt.Sprintf("The template contained an object type unknown to `oc new-app`.  Use `oc process -f %s | oc create -f -` instead.  Error details: %v", config.ComponentInputs.TemplateFiles[0], err)
+			case len(config.ComponentInputs.Components) > 0:
+				useProcessMsg = fmt.Sprintf("The template contained an object type unknown to `oc new-app`.  Use `oc process -f %s | oc create -f -` instead.  Error details: %v", config.ComponentInputs.Components[0], err)
+			default:
+				useProcessMsg = fmt.Sprintf("The template contained an object type unknown to `oc new-app`.  Use `oc process -f <template> | oc create -f -` instead.  Error details: %v", err)
```
Comment (soltysh) [inline pkg/oc/cli/newapp/newapp.go]: I agree with QA, but I wouldn't go as far as having multiple suggestions here. Just this one is sufficient.

Comment (gabemontero) [reply pkg/oc/cli/newapp/newapp.go]: I'm interpreting what you as saying @soltysh as we should not bother trying to substitute the actual parameter the provided as part of the message

I'll update in a bit, but of course clarify further as needed.

Comment (soltysh) [reply pkg/oc/cli/newapp/newapp.go]: Yup, just `Use oc process -f <template> | oc create -f -` is sufficient in this case.

Comment (gabemontero) [reply pkg/oc/cli/newapp/newapp.go]: ok cool thx

**Comment by gabemontero** [comment]
comment from @soltysh addressed

@soltysh @adambkaplan ptal

**Comment by gabemontero** [comment]
`Unauthorized` on latest e2e-aws

**Pull request merged**